### PR TITLE
[1737] Update new features guidance

### PR DESCRIPTION
**Pull request by davidgisbey**
### Context

The new features does not currently give a detailed forecast for future feature roll out

### CODE REVIEW

File: app/views/pages/new_features.html.erb [hunk 1]
```diff
@@ -10,24 +10,33 @@
...
-    <h2 class="govuk-heading-l">May to June 2019</h2>
-    <p class="govuk-body">You’ll be able to:</p>
+    <h2 class="govuk-heading-l">July 2019</h2>
+    <p class="govuk-body">In July we’ll roll over your courses. Once we’ve done this, you’ll be able to:</p>
     <ul class="govuk-list govuk-list--bullet">
-      <li>edit the UCAS Apply eligibility requirements for a course</li>
+      <li>view your courses for the next cycle (2020/21)</li>
+      <li>edit the contents of those courses</li>
+      <li>review their basic details</li>
+      <li>special educational needs and disabilities (SEND) information</li>
```
Comment (fofr) [inline app/views/pages/new_features.html.erb]: > "Once we’ve done this, you’ll be able to special educational needs and disabilities (SEND) information"
This doesn't read right. I'd drop this line altogether.

File: app/views/pages/new_features.html.erb [hunk 2]
```diff
@@ -10,24 +10,33 @@
...
+      <li>request a new course using a Google Form </li>
     </ul>
 
-    <h2 class="govuk-heading-l">May to June 2019</h2>
-    <p class="govuk-body">You’ll be able to:</p>
+    <h2 class="govuk-heading-l">July 2019</h2>
+    <p class="govuk-body">In July we’ll roll over your courses. Once we’ve done this, you’ll be able to:</p>
     <ul class="govuk-list govuk-list--bullet">
-      <li>edit the UCAS Apply eligibility requirements for a course</li>
+      <li>view your courses for the next cycle (2020/21)</li>
```
Comment (fofr) [inline app/views/pages/new_features.html.erb]:
```suggestion
      <li>view your courses for the next cycle (2020 – 2021)</li>
```

File: app/views/pages/new_features.html.erb [hunk 3]
```diff
@@ -10,24 +10,33 @@
...
+      <li>edit the contents of those courses</li>
+      <li>review their basic details</li>
+      <li>special educational needs and disabilities (SEND) information</li>
     </ul>
 
-    <h2 class="govuk-heading-l">July to September 2019</h2>
-    <p class="govuk-body">You’ll be able to:</p>
+    <h2 class="govuk-heading-l">August to September 2019</h2>
+    <p class="govuk-body">You’ll be able to edit all the basic details of your 2020/21 courses, including:</p>
```
Comment (fofr) [inline app/views/pages/new_features.html.erb]:
```suggestion
    <p class="govuk-body">You’ll be able to edit all the basic details of your courses, including:</p>
```
The edit features won't be specific to rollover.

File: app/views/pages/new_features.html.erb (hunk 1 above)
Comment (davidgisbey) [reply app/views/pages/new_features.html.erb]: Have dropped it.

File: app/views/pages/new_features.html.erb (hunk 3 above)
Comment (davidgisbey) [reply app/views/pages/new_features.html.erb]: Actioned. 👍

File: app/views/pages/new_features.html.erb (hunk 2 above)
Comment (davidgisbey) [reply app/views/pages/new_features.html.erb]: And actioned. Thanks Paul 👍

**Pull request merged**